code/FromReferencesTask.cs

```diff
@@ -54,29 +64,39 @@ public override Boolean Execute()
 
 				Directory.CreateDirectory(LibraryDirectoryFullPath);
 
-				var typeScriptFullPaths = GetReferencedProjects(project)
-					.NotNullToNull()
-					.Skip(1)
-					.SelectMany(x => GetTypeScriptItems(x).NotNullToNull(), GetTypeScriptFileFullPath)
-					.NotNull();
+				IEnumerable<ProjectRootElement> referencedProjects = GetReferencedProjects(project)
+									.NotNullToNull()
+									.Skip(1);
```

Comment:
Style: chained method calls should be intended one more level from previous line.